composer.json

```diff
@@ -72,6 +72,9 @@
             "webdriver-binary/binary-chromedriver": true
         },
         "discard-changes": true,
+        "platform": {
+            "php": "7.4"
```

Comment:
This prevents Dependabot (or anyone else for that matter) from accidentally installing dependencies that won't run in PHP 7.4

Reply:
A change like this should have its own PR (and discussion 🙂). Dependabot can just specify its PHP version in its YAML file, right?

Reply:
No, as far as I know this is the only way to constrain the PHP version.

Why would you not want this? If you install a PHP 8 dependency ORCA's tests will fail (as happened to me on #277 initially, what a pain). Limiting packages to those that support PHP 7.4 can only be a good thing.

Reply:
@danepowell : ORCA needs to be installable in php 8.1 as well, so can we remove the platform config mentioned above. 

Also as we need to install ORCA in multiple php versions (7.4, 8.0 and 8.1) we may not pin our dependencies to a specific php version. We need to depend on CI finding it and what happened to you is exactly what we want to happen.